sklearn/cluster/hierarchical.py

```diff
@@ -888,6 +888,10 @@ class Ward(AgglomerativeClustering):
         when varying the number of clusters and using caching, it may
         be advantageous to compute the full tree.
 
+    pooling_func : callable, default=np.mean
+        This combines the values of agglomerated features into a single
```

Comment:
This parameter shouldn't exist for `Ward`. I suspect it's only here as a cheat for `WardAgglomeration`.